android/src/main/kotlin/com/example/flutterimagecompress/core/CompressFileHandler.kt

```diff
@@ -57,14 +54,15 @@ class CompressFileHandler(private val call: MethodCall, result: MethodChannel.Re
         minHeight = tmp
       }
       val targetRotate = rotate + exifRotate
-
+      val outputStream = ByteArrayOutputStream()
```

Comment:
The close method of this class is invalid, and of course it does not matter if it is called.

https://docs.oracle.com/javase/8/docs/api/java/io/ByteArrayOutputStream.html